src/models/User.js

```diff
@@ -0,0 +1,60 @@
+const mongoose = require("mongoose");
+const bcrypt = require("bcryptjs");
+
+const UserSchema = new mongoose.Schema({
+  firstName: {
+    type: String,
+    required: [true, "Please Provide Your First Name"],
+    maxlength: 50,
+    minlength: 3,
+  },
+  lastName: {
+    type: String,
+    required: [true, "Please Provide Your Last Name"],
+    maxlength: 50,
+    minlength: 3,
+  },
+  email: {
+    type: String,
+    required: [true, "Please Provide Email"],
+    match: [
+      /^([\w-\.]+@([\w-]+\.)+[\w-]{2,4})?$/,
+      "Please Provide a Valid Email",
+    ],
+    unique: true,
+  },
+  password: {
+    type: String,
+    required: [true, "Please provide password"],
```

Comment:
add min lenght for password  minlength: [6, "Password must be at least 6 characters"],

Reply:
added